taiga/webhooks/tasks.py

```diff
@@ -95,10 +95,16 @@ def _send_request(webhook_id, url, key, data):
 
     with requests.Session() as session:
         try:
-            response = session.send(prepared_request)
+            response = session.send(prepared_request, allow_redirects=settings.WEBHOOKS_ALLOW_REDIRECTS)
+
+            if not settings.WEBHOOKS_ALLOW_REDIRECTS and response.status_code in [301, 302, 303, 307, 308]:
```

Comment:
Nice, but I would change the order of the conditionals for better legibility:
If the response is a redirection, and their are not allowed...

Reply:
For me, in addition to being more efficient, the order it has is more readable.

It's easier to interpret `not settings.WEBHOOKS_ALLOW_REDIRECTS` than `response.status_code in [301, 302, 303, 307, 308]` because in the first condition you only have to read and in the second you have to know what those numbers mean.